### DISCUSSION TITLE
The address about the trivy access

### DISCUSSION THREAD

**Discussion by loxiaopohaiol** [aquasecurity/trivy#1399] [Q&A] [answered]
I want to use the trivy in my private harbor，but my env doesn't access the internet, so I want to know which website did the trivy access? After I get the website, I maybe open the firewall rule in my env.

**Comment by alexanderkiel** [comment]
You could use tools like `ngrep` or `tcpdump` to discover the servers accessed.

**Comment by knqyf263** [comment, marked as answer]
You can see the list here.
https://aquasecurity.github.io/trivy/latest/getting-started/troubleshooting/#error-downloading-vulnerability-db

**Reply by loxiaopohaiol** [reply]
ok，Thank you very much，I will apply these firewall rule.
